.kokoro/autorelease.sh

```diff
@@ -79,13 +79,17 @@ for pkg in ./NuPkgs/Generated/*.nupkg; do
 done
 
 # Push changes to git, not to the main branch but to branchname
-git add -A
-git commit -m "Update discovery documents and generated code" -m "automatically_generated_update"
-git push --set-upstream origin $branchname
+echo "Start git add"
```

Comment:
Rather than these echoes, possibly change line 3 to "set -ex"? That way the log contains everything that's been done.